docs/running_rune_with_occlum.md

```diff
@@ -70,15 +77,15 @@ docker build . -t ${Occlum_application_image}
 ## Install Inclavare Containers binary
 Download the binary release from [here](https://github.com/alibaba/inclavare-containers/releases/).
 
-### Install `sgx_linux_x64_sdk_2.9.101.2.bin`
-Type the following commands to install `sgx_linux_x64_sdk_2.9.101.2.bin` on your host system.
+### Install SGX SDK
+Type the following commands to install SGX SDK on your host system.
 ```shell
 yum install -y make
 echo -e "no\n/opt/intel\n" | ./sgx_linux_x64_sdk_2.9.101.2.bin
 ```
 
-### Install `sgx_linux_x64_psw_2.9.101.2.bin` 
-Type the following commands to install `sgx_linux_x64_psw_2.9.101.2.bin` on your host system.
+### Install SGX PSW
+Type the following commands to install SGX PSW on your host system.
 ```shell
 yum install -y https://cbs.centos.org/kojifiles/packages/protobuf/3.6.1/4.el7/x86_64/protobuf-3.6.1-4.el7.x86_64.rpm
 ./sgx_linux_x64_psw_2.9.101.2.bin
```

Comment:
Promise me dropping providing sex sdk and paw installer in next 0.4.0 release plz.